src/bot.py

```diff
@@ -33,8 +33,8 @@ def __init__(self, screen: Screen, pick_corpose_on_start: bool = False):
         self._game_stats = GameStats()
         self._game_recovery = GameRecovery(self._screen)
         self._template_finder = TemplateFinder(self._screen)
-        self._item_finder = ItemFinder()
-        self._ui_manager = UiManager(self._screen, self._template_finder)
+        self._item_finder = ItemFinder(self._config)
+        self._ui_manager = UiManager(self._screen, self._template_finder,self._config)
```

Comment:
```suggestion
        self._ui_manager = UiManager(self._screen, self._template_finder, self._config)
```
there are a few more cases of missing whitespace